apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,27 @@
+#include "TutorialServo.h"
+#include <algorithm>
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees = 180.0,
+						float minPulsewidthInMs = 1, float maxPulsewidthInMs = 2)
+	:	m_servoPwmOut(servoPin),
+		m_servoRangeInDegrees(servoRangeInDegrees),
+		m_minPulsewidthInMs(minPulsewidthInMs),
+		m_maxPulsewidthInMs(maxPulsewidthInMs) {}
+
+void TutorialServo::setPositionInDegrees(const float degrees) {
+	float validated_angle = max(0.0, min(servoRangeInDegrees, degrees));

```

Comment:
This is fine, although imo `std::clamp` would be more elegant. https://en.cppreference.com/w/cpp/algorithm/clamp